src/widget/container/mod.rs

```diff
@@ -60,28 +64,45 @@ where
         Self { options }
     }
 
+    fn main_axis(&self) -> Axis {
+        self.options.axis
+    }
+
+    fn cross_axis(&self) -> Axis {
+        self.options.axis.opposite()
+    }
+
     fn content_size(&self) -> IntrinsicSize {
         self.options
             .children
             .iter()
             .fold(IntrinsicSize::none(), |size, widget| {
                 let widget_size = widget.intrinsic_size();
 
-                let width = match (size.width, widget_size.width) {
-                    (Some(width), Some(widget_width)) => Some(width.max(widget_width)),
-                    (Some(width), None) => Some(width),
-                    (None, Some(widget_width)) => Some(widget_width),
+                let cross_axis_size = match (
```

Comment:
I don't love overloading the word "Size" to mean both a width+height and a u32, but I'm not sure what the term is that means "either width or height, whichever is relevant to some axis".

Reply:
Dimension?